### PR TITLE
Randomizezd location via coordintes added.

### PR DESCRIPTION
**Pull request by rj1061**
Changed Topology according to data passes.

### CODE REVIEW

**Comment by psk285** [comment]
Reviewed

File: topologies/wordcount.py [hunk 1]
```diff
@@ -14,5 +14,5 @@ class WordCount(Topology):
     yelp_spout = YelpSpout.spec()
     count_bolt = WordCountBolt.spec(inputs={word_spout: Grouping.fields('word')},
                                     par=2)
-    new_count_bolt = WordCountBolt.spec(inputs={yelp_spout: Grouping.fields('something')},
-                                        par=2)
\ No newline at end of file
+    new_count_bolt = WordCountBolt.spec(inputs={yelp_spout: Grouping.fields('searchJson')},
+                                        par=2)
```
Comment (rj1061) [inline topologies/wordcount.py:18]: :+1:

Comment (maqzi) [reply topologies/wordcount.py:18]: :+1:

**Comment by muraleo** [comment]
i'm good

File: topologies/wordcount.py (hunk 1 above)
Comment (muraleo) [reply topologies/wordcount.py:18]: It's good.

**Review by muraleo — APPROVED** [review]

**Pull request merged**